rpcs3/Emu/Cell/lv2/sys_fs.cpp

```diff
@@ -207,6 +207,7 @@ lv2_fs_mount_point* lv2_fs_object::get_mp(std::string_view filename, std::string
 {
 	auto result = &g_mp_sys_no_device; // Default fallback
 	const auto mp_name = get_device_path(filename);
+	std::string_view path_fix;
```

Comment:
```suggestion
	std::string path_fix;
```

Reply:
Done.

Reply:
Because a stringview is not supposed to be